docs/basic-features/fast-refresh.md

```diff
@@ -105,6 +105,4 @@ screen!
 
 Sometimes, this can lead to unexpected results. For example, even a `useEffect`
 with an empty array of dependencies would still re-run once during Fast Refresh.
-However, writing code resilient to occasional re-running of `useEffect` is a
-good practice even without Fast Refresh. This makes it easier for you to later
-introduce new dependencies to it.
+However, writing code resilient to occasional re-running of useEffect is a good practice even without Fash Refresh. It will make it easier for you to introduce new dependencies to it later on and it's enforced by [React Strict Mode](/docs/api-reference/next.config.js/react-strict-mode), which we highly recommend enabling.
```

Comment:
My suggestion is still different to what you changed, I added a new line and used multiple lines for the paragraph, due to this section being a bit different in the way it's written compared to others.

```suggestion

However, writing code resilient to occasional re-running of useEffect is a good practice even
without Fash Refresh. It will make it easier for you to introduce new dependencies to it later on 
and it's enforced by [React Strict Mode](/docs/api-reference/next.config.js/react-strict-mode),
which we highly recommend enabling.
```